src/MLJDecisionTreeInterface.jl

```diff
@@ -71,9 +81,24 @@ function MMI.fit(m::DecisionTreeClassifier, verbosity::Int, X, y)
     fitresult = (tree, classes_seen, integers_seen, features)
 
     cache  = nothing
-    report = (classes_seen=classes_seen,
-              print_tree=TreePrinter(tree),
-              features=features)
+
+    # generate feature importances for report
+    if m.feature_importance != :none
+        fi = feature_importance_options[m.feature_importance](tree)
+        fi_pairs = collect(Dict(zip(features, fi)))
+        # sort descending
+        sort!(fi_pairs, by= x->-x[2])
+
+        report = (classes_seen=classes_seen,
+                  print_tree=TreePrinter(tree),
+                  features=features,
+                  feature_importances=fi_pairs)
+    else
+        report = (classes_seen=classes_seen,
+                  print_tree=TreePrinter(tree),
+                  features=features,
+                  )
+    end
```

Comment:
The computation of feature importances should be put in the `MMI.feature_importances` method, see above.